assets/webpack.config.js

```diff
@@ -16,7 +16,7 @@ module.exports = (env, options) => ({
     ]
   },
   entry: {
-    "./js/app.js": ["./js/app.js"].concat(glob.sync("./vendor/**/*.js"))
+    "./js/app.js": ["./js/app.js"]
```

Comment:
Strings must use singlequote.

[Read more about it here.](http://eslint.org/docs/rules/quotes)